src/cli.rs

```diff
@@ -20,11 +24,13 @@ pub fn bootstrap_cli() {
     let local_binary_project_directory = per_project_directory.join(LOCAL_BINARY_PROJECT_NAME);
     let path_to_local_release_binary =
         local_binary_project_directory.join(format!("target/release/{LOCAL_BINARY_PROJECT_NAME}"));
-    if should_regenerate_local_binary(&config_file_path, &path_to_local_release_binary) {
+    let command_line_args = env::args_os().collect::<Vec<_>>();
+    let args = Args::parse_from(command_line_args.iter().cloned());
+    if should_regenerate_local_binary(&config_file_path, &path_to_local_release_binary, &args) {
```

Comment:
I added a `--force-rebuild` flag to `tree-sitter-lint` (since during local development the `tree_sitter_lint` dependency (of the "local custom binary") itself may have changed so it's not sufficient to just rely on the config file timestamp comparison)

Which means we need to parse the command-line args both here at the "shell" layer and still forward them along to get parsed by the local custom binary